chalicelib_fourfront/checks/wrangler_checks.py

```diff
@@ -540,6 +561,15 @@ def biorxiv_version_update(connection, **kwargs):
     else:
         check.status = 'PASS'
         check.summary = check.description = 'All current bioRxiv Publications are up to date'
+
+    if problem_ids:
+        check.status = 'WARN'
+        check.summary = check.summary + f"\nThere are {len(problem_ids)} misformatted or problematic doi pub IDs"
```

Comment:
For some reason, the newline shows up as just `\n` in the foursight check output (I noticed it in the doppelgänger check output). Okay to omit?

Reply:
I don't think it will be added as a new line so might look funky but it already does anyway so removed \n